frontend/src/static/js/v_authorship.js

```diff
@@ -40,21 +41,40 @@ window.vAuthorship = {
       totalBlankLineCount: '',
       filesSortType: 'lineOfCode',
       toReverseSortFiles: true,
-      activeFilesCount: 0,
+      isActive: true,
       filterSearch: '*',
       sortingFunction: (a, b) => -1 * window.comparator(filesSortDict.lineOfCode)(a, b),
-      isSearchBar: false,
-      isCheckBoxes: true,
     };
   },
 
   watch: {
+    selectedFileTypes() {
+      if (this.selectedFileTypes.length === this.fileTypes.length) {
+        this.isSelectAllChecked = true;
```

Comment:
Can make it a computed value?

Reply:
Here the new logic is: 1. untick the all if we untick ALL other file type check box. 2. tick `all` if we manually tick all other file type. For this reason, it is impossible to make it a computed value.

Reply:
OK, got your point.